### DISCUSSION TITLE
Scrollbar

### DISCUSSION THREAD

**Discussion by ghost** [skydoves/PowerSpinner#74] [Q&A]
Guys, I may be doing something wrong, but my scroll bar is not showing when I set values, no matter a lot, a little, nothing.

I followed this link (https://github.com/skydoves/PowerSpinner/issues/21), I looked at the library sources, but as I understand it should work out of the box without additional. settings, where did I do something wrong?

I use a standard string adapter, I don’t do any additional settings for the component, everything I tried didn’t work.

@skydoves, help me, please.

**Comment by ghost** [comment]
Maybe someone help, if you need add scrollbar, but don't need a divider, just add this code

            app:spinner_divider_show="true"
            app:spinner_divider_size="0dp"

**Comment by skydoves** [comment]
Hi @umbrellait-dmitry-volkov,
Thanks for reporting your issue :)
In my device, I can see the scrollbar but the colour is really dark (almost hard to find it).
Could you check using the demo project with some changes?

```gradle
app:spinner_popup_height="170dp" // smaller height size than the original height
```
